src/core/prompts/system.ts

```diff
@@ -288,33 +291,16 @@ Array of options here (optional), e.g. ["Option 1", "Option 2", "Option 3"]
 <replace_in_file>
 <path>src/components/App.tsx</path>
 <diff>
-<<<<<<< SEARCH
```

Comment:
this made it look like there were changes in a rebase. i'll get this all back into the code